engine/gamesys/src/gamesys/gamesys_private.cpp

```diff
@@ -91,39 +98,57 @@ namespace dmGameSystem
                     out_desc.m_ElementIds[3] = element_ids[3];
                 }
 
-                if (value != 0x0 && use_value_ptr)
+                if (value != 0x0)
```

Comment:
This was actually a 4 year old bug, that made it impossible to get the value you set for the component's overridden render constant (it always returned the material constant instead).